code/modules/projectiles/guns/projectile/automatic/luty.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/gun/projectile/automatic/luty
+    name = "handmade SMG .35 Auto \"Luty\""
+    desc = "A dead simple open-bolt automatic firearm, easily made and easily concealed.\
+            A gun that has gone by many names, from the Grease gun to the Carlo to the Swedish K. \
+            Some designs are too good to change."
+    icon = 'icons/obj/guns/projectile/luty.dmi'
+    icon_state = "luty"
+    item_state = "luty"
+
+    w_class = ITEM_SIZE_NORMAL
+    can_dual = TRUE
+    caliber = CAL_PISTOL
+    slot_flags = SLOT_BELT|SLOT_HOLSTER
+    ammo_type = /obj/item/ammo_casing/pistol
+    load_method = MAGAZINE
+    mag_well = MAG_WELL_PISTOL|MAG_WELL_H_PISTOL|MAG_WELL_SMG
+    magazine_type = /obj/item/ammo_magazine/smg
+
+    init_firemodes = list(
+        FULL_AUTO_400,
+        SEMI_AUTO_NODELAY,
+        )
+
+    can_dual = 1
+    damage_multiplier = 0.7
+    penetration_multiplier = 0.9
+    recoil_buildup = 1
+    one_hand_penalty = 5 //SMG level.
+    spawn_blacklisted = TRUE
+
+    origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2)
+    matter = list(MATERIAL_STEEL = 15, MATERIAL_WOOD = 10)
+    gun_parts = list(/obj/item/part/gun = 2 ,/obj/item/stack/material/steel = 15)
```

Comment:
Remove gun parts from the list , leave only the steel in. if it doesn't have plasteel in crafting it is not meant to give gun parts.

Reply:
it's got gun parts in the crafting, that's good enough for me

Reply:
fair enough , didn't notice.